docs/component.md

```diff
@@ -457,103 +472,18 @@ const myGreetingInstance = new MyGreeting({
     subject: "friend"
   }
 });
-// myGreetingInstance.element is <my-greeting>Hello friend</my-greeting>
-// myGreetingInstance.viewModel has {subject: "friend"}
+
+myGreetingInstance.element; // is <my-greeting>Hello friend</my-greeting>
+
+myGreetingInstance.viewModel; // is MyGreeting.ViewModel{subject: "friend"}
```

Comment:
Check out the changes in this section on the 5.0 site: https://canjs.github.io/next/doc/can-component.html#Programmaticallyinstantiatingacomponent